src/core/environment.js

```diff
@@ -328,6 +328,23 @@ p5.prototype.setFrameRate = function(fps) {
   return this.frameRate(fps);
 };
 
+/**
+ * Returns the current _targetFrameRate variable.
+ * @method getTargetFrameRate
+ * @return {Number} current _targetFrameRate
+ * @example
+ * <div><code>
+ * function draw() {
+ *   frameRate(20);
+ *   console.log("frameRate: ", frameRate())
+ *   console.log("targetFrameRate: ", getTargetFrameRate());
```

Comment:
Can these be displayed without using `console.log`? As it is not common to have the browser console open while viewing the documentation. You can do something similar to `frameCount()`'s example perhaps?

Reply:
Yes, will work on that.